Tools/LyTestTools/ly_test_tools/o3de/asset_processor.py

```diff
@@ -1089,5 +1089,5 @@ def get_num_failed_processed_assets(output):
     return -1 if not result else int(result)
 
 
-def has_invalid_server_address(output):
-    return parse_output_value(output, 'Invalid server address') is not None
+def has_invalid_server_address(output, serverAddress):
+    return parse_output_value(output, f'Server address ({serverAddress}) is invalid! Reverting back to ()') is not None
```

Comment:
This is an extremely brittle check on the entire specific failure message, where most any change will require updating this file. It also reuses a fairly inefficient parsing function. I recommend instead using:
```python
address_invalid_warning = f"({serverAddress}) is invalid"
for line in output:
    if address_invalid_warning in line:
        return True
return False
```